reference/resources/address.yaml

```
@@ -1,8 +1,7 @@
 ---
 title: Address
 type: object
-description: |-
-  The billing address of the buyer.
+description: The address of the buyer.
```

```suggestion
description: An address for the buyer.
```
Not sure, just in the case of shippingAddress which uses the same resource, the buyer can have more than one so "the" doesn't entirely fit.